src/commands/insult.ts

```diff
@@ -0,0 +1,69 @@
+import { Client, Message } from 'discord.js';
+import axios from 'axios';
+
+class Users {
+    to: string;
+    from: string;
+    constructor(to: string, from: string){
+        this.to = to;
+        this.from = from;
+    }
+}
+
+class Variables {
+    users: Users;
+    constructor(users: Users){
+        this.users = users;
+    }
+}
+
+class GraphRequest {
+    variables: Variables;
+    operationName: string;
+    query: string;
+    constructor(variables: Variables, operationName: string,  query: string){
+        this.variables = variables;
+        this.operationName = operationName;
+        this.query = query;
+    }
+}
+   
+exports.help = {
+    description: 'Use this to insult people, call them names, and make them cry.',
+    name: 'Insult',
+    usage: 'insult'
+};
+
+exports.run = async(client: Client, msg: Message, args: string[]) => {
```

Comment:
Super minor thing, can we add a space after the word 'async'? Not having one implies that this is the name of the function at first glance, but that might just be me